### PR TITLE
Update commit printing

### PR DESCRIPTION
**Pull request by MDrakos**
https://www.pivotaltracker.com/story/show/175487029

### CODE REVIEW

File: internal/table/table.go [hunk 1]
```diff
@@ -175,7 +175,12 @@ func renderRow(providedColumns []string, colWidths []int) string {
 			}
 
 			suffix := strings.Repeat(" ", maxLen-end)
-			result += pad(string(colValue[0:end]) + suffix)
+			if len(colValue) > end && colValue[end-1] == linebreakRune {
+				result += pad(string(colValue[0:end-1]) + suffix)
+			} else {
+				result += pad(string(colValue[0:end]) + suffix)
+			}
+
```
Comment (MDrakos) [inline internal/table/table.go:183]: I'm unsure if this is the correct way to handle this, though it did fix my problem. Please let me know if there is a better approach

File: pkg/cmdlets/commit/commit.go [hunk 2]
```diff
@@ -13,26 +13,62 @@ import (
...
+	}, nil
+}
+
+func shortHash(commitID string) string {
+	split := strings.Split(commitID, "-")
+	if len(split) == 0 {
+		return ""
+	}
+	return split[0]
+}
```
Comment (MDrakos) [inline pkg/cmdlets/commit/commit.go:69]: I had to use a short hash as the formatting can be a little odd when we use the full hash. I will file a bug with details of the output with long values in the commit and message columns

Comment (MDrakos) [reply pkg/cmdlets/commit/commit.go:69]: Story here: https://www.pivotaltracker.com/story/show/175863427

File: internal/table/table.go (hunk 1 above)
Comment (mdrohmann) [reply internal/table/table.go:183]: Maybe we could do something like this:

```
			skipLineBreak := 0
			if breakpos := runeSliceIndexOf(colValue, linebreakRune); breakpos != -1 && breakpos < end {
				end = breakpos + 1
				skipLineBreak = 1
			}
 			suffix := strings.Repeat(" ", maxLen-end)
 			result += pad(string(colValue[0:end-skipLineBreak]) + suffix) 

```

Comment (mdrohmann) [reply internal/table/table.go:183]: That would get rid of the second if-clause

File: internal/table/table_test.go [hunk 3]
```diff
@@ -209,7 +209,7 @@ func Test_renderRow(t *testing.T) {
 				colWidths:       []int{8, 8},
 			},
 			"  abcd    abcd  \n" +
-				"          e\n    \n" +
```
Comment (mdrohmann) [inline internal/table/table_test.go:212]: Well, it looks like this unit-test was broken all along.  Sorry about that.

**Review by mdrohmann — CHANGES_REQUESTED** [review]

File: internal/table/table_test.go (hunk 3 above)
Comment (MDrakos) [reply internal/table/table_test.go:212]: No worries, and thank you for confirming that this was actually broken. I thought the two line breaks seemed a little suspect

**Review by mdrohmann — APPROVED** [review]

**Pull request merged**